### PR TITLE
Jt/support primary app id on install

### PR DESCRIPTION
**Pull request by Jtang-1**
### Migrations

- [ ] The migrations in this code can be safely applied first independently of the code.

### Special deploy instructions

- [ ] This PR can be deployed after merge with no further considerations.

### Rollback instructions

- [ ] This PR can be reverted after deploy with no further considerations.

### Review

- [ ] The set of people pinged as reviewers is appropriate for the level of risk of the change.

### CODE REVIEW

File: src/main/java/org/commcare/formplayer/sqlitedb/ApplicationDBPath.java [hunk 1]
```diff
@@ -11,17 +11,23 @@ class ApplicationDBPath extends DBPath {
...
         this.appId = appId;
+        this.appVersion = appVersion;
     }
 
     @Override
     public String getDatabasePath() {
-        return DbUtils.getDbPathForUser(domain, username, asUsername) + File.separator + appId;
+        String path = DbUtils.getDbPathForUser(domain, username, asUsername) + File.separator + appId;
+        if (appVersion != null) {
+            path = path + "_" + appVersion;
```
Comment (shubham1g5) [inline src/main/java/org/commcare/formplayer/sqlitedb/ApplicationDBPath.java:30]: optional: Seems like we are doing this concatenation of `appId +  "_" + appVersion` at multiple places in the PR and think would be nice to have this as a util method somewhere and reuse it.

File: src/test/resources/requests/delete_db/delete_db.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "app_id": "casetestappid",
   "username": "casetestuser",
-  "domain": "casetestdomain"
+  "domain": "casetestdomain",
+  "appVersion": null
```
Comment (shubham1g5) [inline src/test/resources/requests/delete_db/delete_db.json:5]: Confirming that tests passes even if we omit `"appVersion": null` from here ?

File: src/main/java/org/commcare/formplayer/objects/SerializableFormSession.java [hunk 3]
```diff
@@ -42,6 +42,9 @@ public enum SubmitStatus {
     @Column(name = "appid", updatable = false)
     private String appId;
 
+    @Column(name = "appversion", updatable = false)
+    private String appVersion;
```
Comment (shubham1g5) [inline src/main/java/org/commcare/formplayer/objects/SerializableFormSession.java:46]: Think it should not but think we should verify that it doesn't break Incomplete forms on resuming after deploy (can be part of QA)

**Review by shubham1g5 — COMMENTED** [review]
Looks quite solid to me overall. Just some questions.  Also can we link spec in the PR description. Thanks!

**Comment by Jtang-1** [comment]
Pivoted to a more simple solution that does not require FP changes ([HQ PR](https://github.com/dimagi/commcare-hq/pull/35013/files/6514a2b493270d989d3904e6c06a58f878f66633..f88b6a7ceb5ce33192c93231635cbd3959db5938)) so closing this.

**Pull request closed without merging**